include/quic/endpoint.hpp

```diff
@@ -41,6 +41,9 @@ namespace oxen::quic
     {
       private:
         void handle_ep_opt(opt::enable_datagrams dc);
+        void handle_ep_opt(opt::outbound_alpns alpns);
+        void handle_ep_opt(opt::inbound_alpns alpns);
+        void handle_ep_opt(opt::handshake_timeout timeout);
```

Comment:
I feel like this might make more sense in `IOContext`, where it would still be propogated across all connections maintained by the Endpoint